Documentation/PackedAssetBuilds.md

```diff
@@ -1,64 +1,126 @@
+
+This guide uses `C:\git\o3de` is the source for the O3DE, cloned from GitHub.
+`C:\git\o3de-multiplayersample` is the location of the clone project. These paths are optional. If you choose to use different paths, amend the command scripts in the rest of the guide accordingly.
+
 ## Windows Profile PAK Setup
 
-### Install WWise
-Go to https://www.audiokinetic.com/download/, create a login, log in, and download the installer. WWise is needed to process and package audio assets in the project.
+Multiplayer Sample uses Wwise gem and assets for audio effects. O3DE engine (and O3DE installers) do not include Wwise support by default. In order to add the Wwise support in the engine, one must first install Wwise SDK and then re-build O3DE engine from source.
 
-![WWise installer options](Media/wwise_installer_options.png)
+### Install Wwise
+Go to https://www.audiokinetic.com/download/, create a login, log in, and download the installer. Wwise is needed to process and package audio assets in the project.
+
+![Wwise installer options](Media/Wwise_installer_options.png)
 
 Inside the installer select the version to use.  Install version **2021.1.11.7933**, select both Authoring and SDK, Microsoft platform.
 
-![WWise version selection](Media/wwise_installer_version_selection.png)
+![Wwise version selection](Media/Wwise_installer_version_selection.png)
 
 > REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
 
-### Build profile build and process assets
+### Build o3de-multiplayersample and the engine (non-monolithic)
+
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
+
+Clone the engine from source.
+
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
+
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build_non_mono`.
+
+```shell
+C:\git\o3de> mkdir build_non_mono
+```
+> The location and the name of the build folder is optional.
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
 
-### Test the profile build
+```shell
+C:\git\o3de> cmake -S . -B build_non_mono -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
+```
```

Comment:
CMake will create the build folder for the user.
So the step to make the directory is not needed.

Also I recommend creating the build folder inside of the `<engine-root>/build` directory as that directory is part of the .gitignore file and will not show up as modifications to the repo.

Reply:
Sounds good, updated